### PR TITLE
A slightly nicer GetNextTuesday-function

### PR DESCRIPTION
**Pull request by code1line**
I believe it at least... :-)

### CODE REVIEW

File: src/live.asp.net/Controllers/AdminController.cs [hunk 1]
```diff
@@ -130,21 +130,8 @@ private void UpdateAdminViewModel(AdminViewModel model, LiveShowDetails liveShow
...
-            else
-            {
-                while (nextTuesday.DayOfWeek != DayOfWeek.Tuesday)
-                {
-                    nextTuesday = nextTuesday.AddDays(1);
-                }
-            }
-
-            return nextTuesday.Date;
+            var dayOfWeek = (int)nowPst.DayOfWeek;
```
Comment (hishamco) [inline src/live.asp.net/Controllers/AdminController.cs:133]:
```csharp
var dayOfWeek = (int) nowPst.DayOfWeek;
```

Comment (hishamco) [reply src/live.asp.net/Controllers/AdminController.cs:133]: Also add a new line between the two statements

**Comment by hishamco** [comment]
@code1line I wrote a simpler implementation for this in PR #101, hope you don't mind

**Comment by DamianEdwards** [comment]
I took the other PR for this.

**Pull request closed without merging**